test/cqlpy/test_describe.py

```diff
@@ -1656,7 +1658,9 @@ def try_describe_with_passwords(role):
             try_describe_with_passwords(r1)
             try_describe_with_passwords(r2)
 
-def test_desc_roles_format(cql):
+# Marked as `scylla_only` because we verify that the output of `DESCRIBE SCHEMA`
+# contains information about auth. That's not the case in Cassandra.
+def test_desc_roles_format(cql, scylla_only):
```

Comment:
This is a pre-existing problem, but something seems very fishy to me about how AuthSLConext works. Instead of just rembering which roles we added and removing them (we actually have in util.py a context manager `new_user()` which does exactly this!), we add a role and then at the end loop over all roles dropping ones that seem "non default".
Would be good to to fix this.